ci/terraform/modules/api-gateway/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "api_gateway_name" {
   type        = string
 }
 
-variable "tags" {
+variable "extra_tags" {
   description = "The tags to apply to the created resources"
```

Comment:
Does `extra_tags` exists as it adds additional tags to the resource on top of the other set of default tags?

Reply:
yeah, that's exactly right.

I'll update the description of the variable to make this more obvious!